include/translated-term.php

```diff
@@ -338,4 +338,81 @@ protected function get_db_infos() {
 			'default_alias' => 't',
 		);
 	}
+
+	/**
+	 * Wrapper of `wp_insert_term` with the language.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string       $term     The term name to add.
+	 * @param string       $taxonomy The taxonomy to which to add the term.
+	 * @param PLL_Language $language The term language.
+	 * @param array|string $args     The taxonomy of the term.
```

Comment:
```suggestion
	 * @param array|string $args     Array or query string of arguments for inserting a term.
```
According to WP doc.